docs/en/enterprise-edition/content-collections/dashboards/dashboards-discovery-exposure-management.adoc

```diff
@@ -36,23 +39,38 @@ Displays a complete view of your cloud assets discovered across your attack surf
+Displays top risks that affect your security posture due to public exposure of assets on the internet. Each row highlights the risks based on categories, and the links in each row will help you explore and gain additional context on the affected assets, along with options to convert and inspect them. The risks are categorized as follows:
+
+* *Critical—* Internet-exposed assets connected to a managed asset on Prisma Cloud, with exploitable vulnerabilities and available patches.
+* *Exploited—* Internet-exposed assets discovered in the last two years, with exploitable vulnerabilities and available patches.
+* *Patchable—* Internet-exposed assets discovered in the last two years, with vulnerabilities and available patches.
+* *Vulnerable—* Internet-exposed assets with vulnerabilities that are susceptible to exploitation.
+* *Exposed—* Assets that can be accessed over the internet.
 
 To take action to secure the assets, you can select the *Convert Unmanaged Assets* button and initiate a bulk conversion of all the unmanaged (exposed) assets to managed (secure) ones simultaneously. Alternatively, you can individually convert assets using the *Convert* button in each row to secure specific assets. This feature is available only for accounts associated with the parent account or organizational unit (OU) accounts on Prisma Cloud. In cases where such mapping is not feasible, you can *Inspect Unmanaged Assets* (issues related to exposed assets) and send an email to your inbox containing the summary of the exposed assets. 
 
 Selecting the *Convert to Managed* button will onboard the assets to Prisma Cloud. Once the unmanaged assets convert to managed assets, Prisma Cloud will monitor these assets for vulnerabilities, misconfigurations, compliance issues, and so on.
 
-In addition to conversion options, the *Inspect Unmanaged Assets* column lists the scanned asset findings, enabling you to review the exposed assets. Select a link to view the side panel. The panel includes a comprehensive list of assets, risk types, and associated cloud providers. Further exploration of each asset provides detailed information such as asset *Overview*, *Exposed Services*, identified *Findings*, and *Network Flow Log* visualizations.
+In addition to conversion options, the *Inspect Unmanaged Assets* column lists the scanned asset findings, enabling you to review the exposed assets. Select a link to view the side panel. The panel includes a comprehensive list of assets, risk types, and associated cloud providers. Further exploration of each asset provides detailed information as follows:
+
+* *Overview—* This is the default tab providing a comprehensive overview of the selected asset, including details such as server name, resource type, expiry date, and so on.
```

Comment:
I'd suggest using etc here

This is the default tab providing a comprehensive overview of the selected asset, including details such as server name, resource type, expiry date etc.